sw/device/tests/otbn/otbn_test_utils.c

```diff
@@ -0,0 +1,67 @@
+// Copyright lowRISC contributors.
```

Comment:
Should we move the entropy utils to a separate `sw/device/tests/entropy/entropy_utils.c` so that even the non OTBN tests will benefit?

Reply:
SG.

Reply:
and Done.